src/Magento/FunctionalTestingFramework/Suite/views/SuiteClass.mustache

```diff
@@ -112,8 +119,38 @@ class {{suiteName}} extends \Codeception\GroupObject
             }
 
             PersistedObjectHandler::getInstance()->clearSuiteObjects();
+
+            $this->closeSession($webDriver);
+
             print sprintf(self::$HOOK_EXECUTION_END, "after");
         }
     }
     {{/after}}
+
+    /**
+     * Close session method closes current session.
+     * If config 'close_all_sessions' is set to 'true' all sessions will be closed.
+     *
+     * @param MagentoWebDriver $webDriver
+     * return void
+     */
+    private function closeSession(MagentoWebDriver $webDriver): void
+    {
+        $webDriverConfig = $webDriver->_getConfig();
+        $webDriver->_closeSession();
+        if (isset($webDriverConfig['close_all_sessions']) && $webDriverConfig['close_all_sessions'] === "true") {
+            $wdHost = sprintf(
```

Comment:
I see a `protected function stopAllSessions()` in `WebDriver.php`, which MagentoWebDriver extends. It seems to do similarly what we do here, did you try and make it work?